recipe/build_base.sh

```diff
@@ -513,4 +515,4 @@ fi
 
 # Workaround for old conda versions which fail to install noarch packages for Python 3.10+
 # https://github.com/conda/conda/issues/10969
-ln -s "${PREFIX}/lib/python3.10" "${PREFIX}/lib/python3.1"
+ln -s "${PREFIX}/lib/python3.11" "${PREFIX}/lib/python3.1"
```

Comment:
Should it be `ln -s ${PREFIX}/bin/python3.11 ${PREFIX}/bin/python3.11`? How is the folder really named if we take into account the conda issue #46  ?

Reply:
Yes this is correct, it will truncate 3.11 into 3.1 in the same way. See root cause [here](https://github.com/conda/conda/blob/33a142c16530fcdada6c377486f1c1a385738a96/conda/common/path.py#L181)